packages/modules/backup_clouds/samba/backup_cloud.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+import logging
+import re
+from smb.SMBConnection import SMBConnection
```

Comment:
Die smb-Lib muss noch installiert werden. Dazu in requirements.txt `pysmb==1.2.9.1` ergänzen.

Reply:
Ich werde mich nach meinem Urlaub nochmal damit befassen. Leider bin ich neu bei allem. Muss ich den pull request zurück ziehen, die Dateien entfernen und erneut pullen oder geht das auch so? Vielen Dank

Reply:
Du kannst die Änderungen in dem bestehenden Branch bei Dir einpflegen und comitten. Diese tauchen dann automatisch hier auf.